### PR TITLE
Fix data in Console API using mdn-bcd-collector

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR uses the data collected from the mdn-bcd-collector project to fix a few data inaccuracies in the Console API.  Results were manually verified for accuracy.

Tests used: https://mdn-bcd-collector.appspot.com/tests/api/Console

### CODE REVIEW

File: api/Console.json [hunk 1]
```diff
@@ -1403,7 +1405,7 @@
               "version_added": "39"
             },
             "ie": {
-              "version_added": "11"
+              "version_added": false
```
Comment (saschanaz) [inline api/Console.json:1408]: I think this is supported.

image

Comment (foolip) [reply api/Console.json]: @vinyldarkscratch did something go wrong with the methodology here? Can the other changes be trusted?

Comment (saschanaz) [reply api/Console.json]: AFAIK IE only defines console functions when DevTools is opened, so maybe that's the cause.

Comment (vinyldarkscratch) [reply api/Console.json]: If I'm correct, this was an issue of testing IE on Windows 7 vs. Windows 10 -- in BrowserStack and SauceLabs, the IE versions are ever so slightly behind on updates.  I've made sure that in the future, the Selenium scripts will always prefer Windows 10 first, and then Windows 7 if Windows 10 isn't possible.

Comment (saschanaz) [reply api/Console.json]: Does that mean IE has different behaviors in Win7 and Win10? Should that be documented here?

Comment (vinyldarkscratch) [reply api/Console.json]: It's less that IE has different behaviors, and more that the test cloud (SauceLabs) does.

Comment (foolip) [reply api/Console.json]: > AFAIK IE only defines console functions when DevTools is opened, so maybe that's the cause.

It sounds like maybe this wasn't the cause here, but I'll keep this in mind when reviewing any IE changes. Where would this be documented, though?

File: api/Console.json [hunk 2]
```diff
@@ -1362,18 +1362,20 @@
             "opera": {
               "version_added": "60"
             },
+            "opera_android": {
+              "version_added": "50"
+            },
             "safari": {
-              "version_added": false,
-              "notes": "See <a href='https://webkit.org/b/186833'>bug 186833</a>."
```
Comment (foolip) [inline api/Console.json:1367]: https://bugs.webkit.org/show_bug.cgi?id=199184 RESOLVED FIXED, nice!

**Review by foolip — APPROVED** [review]

**Pull request merged**